tutorials/marble/version-control.md

```diff
@@ -0,0 +1,354 @@
+    - [Staging a Commit](#stage-gui)
+    - [Making a Commit](#commit-gui)
+    - [Pushing a Commit](#push-gui)
+    - [Creating a Branch](#create-branch-gui)
+  - [Using Git with the Terminal](#using-git-terminal)
+    - [Staging a Commit](#stage-terminal)
+    - [View Staged Files](#view-stage-terminal)
+    - [Making a Commit](#commit-terminal)
+    - [Pushing a Commit](#push-terminal)
+    - [Creating a Branch](#create-branch-terminal)
+- [Creating a Pull Request](#create-pr)
+- [Merging Your Branch](#merge-branch)
+
+## <a id="intro-git"></a>Introduction to Git
+
+Git is one of the most popular version control systems in use today. It has a native commandline interface but also has 
+a desktop GUI client for Windows and MacOS.  The online GitHub website allows you to store your repository online and access it anywhere with a free account. 
+
+## <a id="best-practices"></a>Git Best Practices
+
+### Organize with Branches
+
+Create a branch for each issue you're working on.  Have one for data ingestion, one for visualization, and one for bug fixes.
+
+This keeps the workflow clean.  If something went wrong you know which branch the changes came from.
+
+### Write Descriptive Commit Messages
+
+A good commit message documents the changes made in the commit so that when looking through the commits you can see the direction your work is going in.
+
+### Commit finished tasks only
+
+Each commit should be a completed logical chunk of the overall workflow.  If you need to commit often, split up the workflow into smaller tasks.
+
+### Test code before committing
+
+Related to only committing finished tasks, test your code to ensure it does what it is intended to do before committing.
+
+## <a id="setup-git"></a>Setting up a Git Repository 
+
+[GitHub Desktop Client](https://desktop.github.com/)
+
+### <a id="github-account"></a>Creating a GitHub Account
+To begin using Git you need to create a free account on [GitHub](https://github.com/).  
+Go to the GitHub website and follow the instructions to create an account. 
+
+### <a id="remote-repository"></a>Creating a Remote Repository
+
+After creating account click `Repository` at the top menu and then `New`
+
+![GitHub Top Menu](images/version-control/github-top-menu.png)
+
+![GitHub New Button](images/version-control/github-new-button.png)
+
+Fill in the form with the details of the repository and click the `Create Repository` button at the bottom of the page.
+
+### <a id="creating-pat"></a>Creating a Personal Access Token
+
+A Personal Access Token is used to authenticate the user when pushing files to the remote repository
+
+Go to [GitHub](https://github.com/) and log into your account.
+
+Click on: **Your User Profile** &rarr; **Settings** &rarr; **Developer Settings** &rarr; **Personal Access Token** &rarr; **Generate New Token or Tokens(classic)**
+
+Choose an expiration date.  Check all the boxes in the form.  Click **Generate Token**. Copy the generated token.
```

Comment:
it may be best to highlight the access token will only be visible once and cannot be viewed again after the single moment of creation

Reply:
Added a note saying the personal access token is only shown at creation.